services/app-api/src/emailer/emails/newQuestionStateEmail.test.ts

```diff
@@ -0,0 +1,270 @@
+import {
+    testEmailConfig,
+    mockContractAmendmentFormData,
+    mockContractOnlyFormData,
+    mockMNState,
+} from '../../testHelpers/emailerHelpers'
+import type { CMSUserType, StateType } from '../../domain-models'
+
+import type { LockedHealthPlanFormDataType } from '../../../../app-web/src/common-code/healthPlanFormDataType'
+import { packageName } from '../../../../app-web/src/common-code/healthPlanFormDataType'
+import { newQuestionStateEmail } from './index'
+
+const defaultSubmitters = ['submitter1@example.com', 'submitter2@example.com']
+
+const flState: StateType = {
+    stateCode: 'FL',
+    name: 'Florida',
+}
+
+const cmsUser: CMSUserType = {
+    id: '1234',
+    role: 'CMS_USER',
+    divisionAssignment: 'DMCO',
+    familyName: 'McDonald',
+    givenName: 'Ronald',
+    email: 'cms@email.com',
+    stateAssignments: [flState],
+}
+
+const dateAsked = new Date('01/01/2024')
+test('to addresses list includes submitter emails', async () => {
+    const sub = mockContractOnlyFormData()
+    const defaultStatePrograms = mockMNState().programs
+    const template = await newQuestionStateEmail(
+        sub,
+        defaultSubmitters,
+        cmsUser,
+        testEmailConfig(),
+        defaultStatePrograms,
+        dateAsked
+    )
+
+    if (template instanceof Error) {
+        console.error(template)
+        return
```

Comment:
I would throw this error if it's an error, that will fail the test. I don't think returning will fail the test but maybe it will b/c there haven't been any assertions? In general, if there's some kind of error in a test I like to throw it to fail. 

Lots of our test helpers throw (see `createAndSubmitTestHealthPlanPackage` for instance)

This is a big exception to our general rule of "return Errors, don't throw" but it's basically the API of jest so it makes sense to me